test/unit/v1.0.0/normalize-v1.test.js

```diff
@@ -1689,6 +1689,49 @@ describe('v1.0.0 normalization', function () {
         });
     });
 
+    describe('retainEmptyValues', function () {
```

Comment:
Can we add the same test for `normalizeSingle` as well? Though the code flow is exactly the same, just to be on the safer side?